src/Features/VisualBasic/Completion/CompletionProviders/CompletionListTagCompletionProvider.vb

```diff
@@ -14,7 +14,8 @@ Namespace Microsoft.CodeAnalysis.VisualBasic.Completion.Providers
         Inherits EnumCompletionProvider
 
         Protected Overrides Function GetPreselectedSymbolsWorker(context As AbstractSyntaxContext, position As Integer, options As OptionSet, cancellationToken As CancellationToken) As Task(Of IEnumerable(Of ISymbol))
-            If context.SyntaxTree.IsObjectCreationTypeContext(position, cancellationToken) Then
+            If context.SyntaxTree.IsObjectCreationTypeContext(position, cancellationToken) OrElse
+                context.SyntaxTree.IsInNonUserCode(position, cancellationToken) Then
```

Comment:
Nitpick: outdent 18 by one char so it doesn't look like it's the first line within the body.